WooCommerce/Classes/ViewRelated/Orders/Order Creation/CustomerSection/OrderCustomerSection.swift

```diff
@@ -75,29 +85,39 @@ struct OrderCustomerSection: View {
                         .footnoteStyle()
                 }
             }
-            if let billingAddressFormatted = viewModel.billingAddressFormatted {
-                addressDetails(title: Localization.billingTitle, formattedAddress: billingAddressFormatted)
-            }
+            .padding([.leading, .bottom, .trailing])
+            .padding(.top, -Layout.linkButtonTopPadding)
             if let shippingAddressFormatted = viewModel.shippingAddressFormatted {
                 addressDetails(title: Localization.shippingTitle, formattedAddress: shippingAddressFormatted)
             }
+            Divider()
+                .padding(.leading)
+            NavigationRow(content: {
+                Text(Localization.showBilling)
+                    .bodyStyle()
+            }, action: {
+                showCustomerSummary.toggle()
```

Comment:
Is this behavior defined in the designs somewhere? I was a little surprised that "Edit" and "View Billing Information" both lead to the summary view. I think I expected it to just show the billing address/contact details (the same as viewing order details for an existing order) but I don't see that actually defined anywhere either way.

I don't think it's a problem for now but I wanted to double check in case we need to bring it up for discussion.

Reply:
Agree, we'll have less confusion with updates to the flow mentioned by Jayson!